### DISCUSSION TITLE
Vanilla-extract with globalStyle

### DISCUSSION THREAD

**Discussion by augustskare** [remix-run/remix#5238] [Q&A]
_I was unable to create an issue, that's why I created a "discussion"._

It's not possible to import a `.css.ts` file with only `globalStyle`. After raising the issue with @markdalgleish directly he pointed out that this looks like a tree shaking issue and it could be solved by updating the `sideEffects` field in your package.json to include `"*.css.ts"`.

[Here is a reproducible example](https://github.com/augustskare/laughing-carnival)

If you'r on the main branch you can see the styles are not being applied. Then if you checkout the "works" branch you can see that it works once you export something (in this example the body variable).

**Comment by markdalgleish** [comment]
Thanks for the bug report! I've opened a PR that fixes this: https://github.com/remix-run/remix/pull/5246

**Reply by markdalgleish** [reply]
PR has now been merged! 🚀
